superbench/benchmarks/model_benchmarks/pytorch_base.py

```diff
@@ -259,3 +289,16 @@ def _cal_params_count(self):
             The count of trainable parameters.
         """
         return sum(p.numel() for p in self._model.parameters() if p.requires_grad)
+
+    def timer(self):
+        """Returns the current time which ensures all previous CUDA events have been finished.
+
+           If there is no GPU present, this defaults to`time.time()`; otherwise it will synchronize
+           CUDA before measuring the time.
```

Comment:
```suggestion
        If there is no GPU present, this defaults to `time.time()`; otherwise it will synchronize
        CUDA before measuring the time.
```

Reply:
updated